client/components/document.vue

```diff
@@ -106,6 +50,16 @@
         this.$subscribe('Document.one', {documentId: this.documentId});
       });
     },
+    methods: {
+      onAvatarClicked(cursor) {
+        this.cursor = cursor;
+      },
+      onEditorScroll() {
+        // we just remove the reference to the previously clicked cursor because all we needed
```

Comment:
When making comments. Please use full sentences. So start with upper-case letter and end with dot. See code style document for more information.